apps/bettafish/src/app/app.component.ts

```diff
@@ -34,7 +34,7 @@ export class AppComponent implements OnInit, AfterViewInit {
     ) {}
 
     ngOnInit(): void {
-        this.appQuery.theme$.pipe(untilDestroyed(this), delay(600)).subscribe(theme => {
+        this.appQuery.theme$.pipe(untilDestroyed(this)).subscribe(theme => {
```

Comment:
Figments added this delay for a reason, so I'd rather not remove it in an unrelated PR.